coops-ui/src/interfaces/informationAlert-interfaces.ts

```diff
@@ -0,0 +1,5 @@
+export interface AlertMessage {
+  title: String
+  msg: String
+  additionalItems: Array<String>
```

Comment:
Use `string` primitive for properties/variables.

You should only use `String` objects to reference the object itself, for example to create a string ("new String()").

See this page https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String under heading "Distinction between string primitives and String objects".